### DISCUSSION TITLE
Say that passwords are "password" for dev instances

### DISCUSSION THREAD

**Discussion by larseggert** [ietf-tools/datatracker#3607] [Ideas]
On the landing and login pages, so people are not surprised.

**Comment by rjsparks** [comment]
I'm not sure that's always the right thing to do, and I'm fairly sure it's _not_ the right thing to do for publicly served instances like the sandbox.

A little bit of "need to have read a document", at least a README, would likely slow down nuisance traffic, and well-intentioned, but off base, vulnerability reports.

I can see having a hint posted in a local development instance being useful.

**Reply by Samric24** [reply]
Yes please and account linked to my mobile

**Comment by rjsparks** [comment]
Speaking of nuisance traffic. Since this isn't an active discussion at this point, I'm closing and locking it.
